addon/templates/components/table.hbs

```diff
@@ -28,14 +28,15 @@
                     (if Row.columnValue.isFixedLeft " data-table-col-fixed-left")
                     (if (eq Row.columnValue this.firstVisibleNonFixedColumn) " data-table-col-first-nonfixed")
                 }}
-                as |columnValue columnMeta|
+                as |columnValue columnMeta tableMeta|
```

Comment:
these internal components from ember-table like `Row.cell` dont yield a `tableMeta` property, so this isnt needed, it would just come directly from the `Table` component, e.g. `this.tableMeta`